spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -215,61 +220,6 @@ def update_info(self, name, kind, position, status, selected):
         self.setSelected(selected)
 
 
-class TreeItem(QTreeWidgetItem):
```

Comment:
Is this class not used anymore anywhere? Just dead code, right?

Reply:
Yep, I checked and we're not using in any other place. Furthermore, I think it was the tree item used in the previous implementation, so we can safely remove it now.